proposals/records.md

```diff
@@ -79,18 +64,60 @@ override Equals(object o) => Equals(o as T);
 
 ### Copy and Clone members
 
-A record type contains two synthesized copying members if methods with the same
-signature are not already declared within the record type:
+A record type contains two synthesized copying members:
 
 * A protected constructor taking a single argument of the record type.
-* A public parameterless instance method called `Clone` which returns the record type.
+* A public parameterless virtual instance "clone" method with a compiler-reserved name
 
 The protected constructor is referred to as the "copy constructor" and the synthesized
-body copies the values of all instance fields declared in the input type to the corresponding 
+body copies the values of all accessible instance fields in the input type to the corresponding
```

Comment:
>all accessible instance fields [](start = 26, length = 30)

Consider clarifying this, obviously all fields within a record are accessible.

Reply:
What about inherited private fields?

Reply:
> What about inherited private fields?

Shouldn't base take care of this?

---
In reply to: [434061428](https://github.com/dotnet/csharplang/pull/3527#discussion_r434061428) [](ancestors = 434061428)

Reply:
Not for the very first record in a hierarchy, inheriting from a user-written class.

Reply:
> Not for the very first record in a hierarchy, inheriting from a user-written class.

That is why I think it would be good to clearly describe two distinct scenarios and what is the behavior of the constructor in each of them.

---
In reply to: [434063198](https://github.com/dotnet/csharplang/pull/3527#discussion_r434063198) [](ancestors = 434063198)